docs/docs/concepts/email-sms.md

```diff
@@ -125,21 +144,57 @@ Hi, please verify your account by clicking the following link:
 Hi, please verify your account by clicking the following link: {{ .VerificationURL }}
 ```
 
-### Custom Headers
```

Comment:
Can you please re-add this?

Reply:
@aeneasr  Sorry, my mistake. I've put it back now.